dev/src/lobster/cubegen.h

```diff
@@ -32,6 +32,7 @@ struct Voxels {
     bool chunks_skipped = false;
     Chunk3DGrid<uint8_t> grid;
     int idx = 0;
+    string name = "";
```

Comment:
no need for the `= ""`, that is the default.

Reply:
ah, with something auto initializing and others not doing that, what is the pattern to avoid failing to empty initialize when depending on that state ?